build.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+set -e
+
+# Install programs required for build
+apk --no-cache add nodejs git curl
+apk --update add tar
+curl -o- -L https://yarnpkg.com/install.sh | sh
```

Comment:
The path is not something I was able to fix unfortunately, I am unable to add it to the path. It does also not really make sense to add it to the path in a docker container.